content/en/building/contact-management/contact-and-users-1.md

```diff
@@ -9,27 +9,27 @@ relatedContent: >
   technical-overview/concepts/db-schema
   building/guides/data/users-bulk-load
 aliases:
-   - /building/tutorials/contact-and-users-1/
-   - /apps/tutorials/contact-and-users-1
+  - /building/tutorials/contact-and-users-1/
+  - /apps/tutorials/contact-and-users-1
 ---
 
 In this tutorial you will learn how to create and edit contacts and their associated users in and application built with the CHT using the default contact creation forms. This will help you get familiar with the UI of the webapp as well as some features and functionality. If you are already comfortable with this, you can skip to [part 2, which covers manipulating contacts and their associated documents using cht-conf]({{% ref "building/contact-management/contact-and-users-2" %}}).
 
 ## Brief Overview of Key Concepts
 
-*Contacts* are people or places that are created in the CHT application.
+_Contacts_ are people or places that are created in the CHT application.
 
-*People* are both patients in the system and users of the system, such as CHWs or Nurses.
+_People_ are both patients in the system and users of the system, such as CHWs or Nurses.
 
-*Places* represent either an actual physical location such as a health facility, clinic, or a grouping such as a household or CHW Area.
+_Places_ represent either an actual physical location such as a health facility, clinic, or a grouping such as a household or CHW Area.
 
-*Contact forms* are forms in the CHT app that are used to create people or places.
+_Contact forms_ are forms in the CHT app that are used to create people or places.
 
-*CHT App Hierarchy* is often modeled after the health system, health program or community structure.  All people who are registered in the app must be associated with a Place. These Places are located in a hierarchy with other Places. For instance, a Family Member is part of a Household. A Household and CHWs are part of a CHW Area. A CHW Area and nurses are part of a Health Facility. Additional levels may be added as needed. The Admin level operates outside of the hierarchy and gives access to all levels and people.
+_CHT App Hierarchy_ is often modeled after the health system, health program or community structure. All people who are registered in the app must be associated with a Place. These Places are located in a hierarchy with other Places. For instance, a Family Member is part of a Household. A Household and CHWs are part of a CHW Area. A CHW Area and nurses are part of a Health Facility. Additional levels may be added as needed. The Admin level operates outside of the hierarchy and gives access to all levels and people.
 
 {{< figure src="app-hierarchy.jpg" link="app-hierarchy.jpg" caption="Default app hierarchy" >}}
 
-*Users* represent credentials and roles / permissions for accessing the application. This can either be:
+_Users_ represent credentials and roles / permissions for accessing the application. This can either be:
```

Comment:
the goal of a PR is to both improve the document and be easy to review.  By changing the italics from `*Users*` to another `_Users_` , you're forcing the review to check what the change is, when in fact it's no change.

like the spacing changes, you're fine to keep this in this PR, but next time, keep PRs with as few changes as possible please!  They'll be easier to review.